ddtrace/vendor/contextvars/__init__.py

```diff
@@ -0,0 +1,160 @@
+import threading
+
+try:
+    from collections.abc import Mapping
+except ImportError:
+    from collections import Mapping
+
+
+__all__ = ("ContextVar", "Context", "Token", "copy_context")
+
+
+_NO_DEFAULT = object()
+
+
+class Context(Mapping):
```

Comment:
This is all really just `ContextVars` API around `threading.local()` it seems, right?

Reply:
yes